pkg/util/trace.go

```diff
@@ -0,0 +1,27 @@
+package util
```

Comment:
Curious, why not just `pkg/trace`?

Reply:
Nothing specific, just thought it was a utility. moving it to `pkg/trace`